C/CJieba/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+using BinaryBuilder, Pkg
+
+name = "CJieba"
+version = v"0.3.0"
+
+sources = [
+    ArchiveSource(
+        "https://github.com/yanyiwu/cjieba/archive/refs/tags/v$(version).tar.gz",
+        "1f5ca82c5d19c38485c8fd50db8711602d339b1e85f15fb864fc116c7f770f12"
+    )
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir $libdir $prefix/share
+curl https://raw.githubusercontent.com/yanyiwu/cppjieba/master/LICENSE > LICENSE
```

Comment:
I'm confused: this is a license for a different project?

Reply:
This repo is the low level C api for cppjieba, the author didn't add a license for this one, but cppjieba use MIT, so it seems appropriate here.

Reply:
I don't want to pesky here, but we shouldn't make assumption about what developers think: they should explicitly state the license they want to use, otherwise we don't have the rights to distribute their software.

Reply:
Good point. I could try to contract the author see if he's will to add a explicit license for cjieba.

Reply:
That'd be perfect, thanks a lot!